### DISCUSSION TITLE
Impossible to deploy my flutter webapp on github.io

### DISCUSSION THREAD

**Discussion by tcrame** [community/community#67971] [Actions]
### Select Topic Area

Bug

### Body

Hello, 

I'm trying to deploy a flutter webapp to my github homepage but i had this error 

```
2s
Run actions/deploy-pages@v2
Artifact exchange URL: https://pipelinesghubeus7.actions.githubusercontent.com/6UoC8bhmg7QcDBYqCkSXAQn6iSbjjR[1](https://github.com/tcrame/tcrame.github.io/actions/runs/6276314151/job/17045964629#step:2:1)a3QEcMiFceGgEqvzWAa/_apis/pipelines/workflows/6276314151/artifacts?api-version=6.0-preview
Creating Pages deployment with payload:
{
	"artifact_url": "https://pipelinesghubeus7.actions.githubusercontent.com/6UoC8bhmg7QcDBYqCkSXAQn6iSbjjR1a3QEcMiFceGgEqvzWAa/_apis/pipelines/1/runs/3/artifacts?artifactName=github-pages&%24expand=SignedContent",
	"pages_build_version": "1dc[9](https://github.com/tcrame/tcrame.github.io/actions/runs/6276314151/job/17045964629#step:2:10)7c4b4908a3f74cb030130957838686eb6ee7",
	"oidc_token": "***"
}
Error: Creating Pages deployment failed
Error: HttpError: Invalid artifact url
    at /home/runner/work/_actions/actions/deploy-pages/v2/webpack:/deploy-pages/node_modules/@octokit/request/dist-node/index.js:86:1
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at createPagesDeployment (/home/runner/work/_actions/actions/deploy-pages/v2/webpack:/deploy-pages/src/internal/api-client.js:[12](https://github.com/tcrame/tcrame.github.io/actions/runs/6276314151/job/17045964629#step:2:13)6:1)
    at Deployment.create (/home/runner/work/_actions/actions/deploy-pages/v2/webpack:/deploy-pages/src/internal/deployment.js:80:1)
    at main (/home/runner/work/_actions/actions/deploy-pages/v2/webpack:/deploy-pages/src/index.js:30:1)
Error: Error: Failed to create deployment (status: 400) with build version 1dc97c4b4908a3f74cb030[13](https://github.com/tcrame/tcrame.github.io/actions/runs/6276314151/job/17045964629#step:2:14)0957838686eb6ee7. Responded with: Invalid artifact url
```

here is the link to the action : https://github.com/tcrame/tcrame.github.io/actions/runs/6276314151/job/17045964629

here is the source of my webapp  : https://github.com/tcrame/tcrame.github.io

I don't know what's happened ....

**Comment by SumanthSShet** [comment]
The error message "Error: Creating Pages deployment failed. Error: HttpError: Invalid artifact url" indicates that GitHub Pages is unable to access the artifact that you are trying to deploy. This could be due to a number of factors, such as:

The artifact URL is incorrect.
The artifact URL is pointing to a file that does not exist.
The artifact URL is pointing to a file that is too large.
The artifact URL is pointing to a file that is in a private repository.
The artifact URL is pointing to a file that is behind a firewall or VPN.
To troubleshoot this error, you can try the following:

Check the artifact URL carefully to make sure that it is correct.
Try uploading the artifact to a different location, such as a public repository or a cloud storage provider.
If the artifact is too large, you can try breaking it up into smaller files and deploying them separately.
If the artifact is in a private repository, you will need to make sure that you have the necessary permissions to access it.
If the artifact is behind a firewall or VPN, you will need to make sure that GitHub Pages can access it.
If you are still having trouble deploying your website to GitHub Pages, you can contact GitHub support for assistance.

In your case, I would recommend checking the artifact URL carefully to make sure that it is correct. You can also try uploading the artifact to a different location, such as a public repository or a cloud storage provider.

Here are some additional tips for deploying a Flutter web app to GitHub Pages:

Make sure that you are using the latest version of the deploy-pages action.
Check the GitHub Pages documentation for information on known issues and workarounds.
Try clearing your GitHub Pages cache.
Try deploying your website from a different computer or network.
If you are still having trouble, you can try searching for help online or contacting GitHub support.

**Comment by tcrame** [comment]
It was a global ssue on github pages deployment.
